modules/serverless-installing-cli-linux-ibm-power-rpm.adoc

```diff
@@ -3,7 +3,7 @@
 // * serverless/installing-kn.adoc
 
 [id="installing-cli-linux-ibm-power-rpm_{context}"]
-= Installing the Knative CLI for Linux on IBM Power Systems using an RPM
+= Installing the Knative CLI (`kn`) for Linux on IBM Power Systems using an RPM
```

Comment:
You need to remove the backticks in all the headings based on the  [repo guidelines](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#assemblymodule-titles-and-section-headings):

> Do not use backticks or other markup in assembly or module headings.

Personally, I'd remove the abbreviation from the heading and add the expanded form and explanation to the first line of text. The ISG says:

> Avoid using abbreviations in headings and titles unless an abbreviation is well known or the spelled-out form is too long.

You're using an acceptable pattern, but check out the rest of the guidance in the "Headings and titles" section.

Whatever changes you make, please be consistent across the PR. :)